backend/app/main.py

```
@@ -14,16 +14,18 @@
 load_dotenv(env_path)
 
 
-def create_app():
+def create_app(config_class=None):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)